octovis/src/ViewerWidget.cpp

```diff
@@ -364,7 +363,6 @@ void ViewerWidget::postDraw(){
   glPushAttrib(GL_ALL_ATTRIB_BITS);
 
   glDisable(GL_COLOR_MATERIAL);
-  qglColor(foregroundColor());
```

Comment:
Is this line intentionally removed?

Reply:
Yes, it it also from the deprecated QGLWidget Class, see https://doc.qt.io/qt-5/qglwidget.html#qglColor, and therefore no longer supported.